framework/styles/theme/default.scss

```diff
@@ -160,6 +160,14 @@ $atomic-default: (
     "selection-bg-disabled": map-get($grey, "lighten-2"),
     "handle-border-disabled": map-get($grey, "lighten-2")
   ),
+  "steps": (
+    "step-icon-bg-visited": transparent,
+    "step-icon-color-visited": map-get($grey, "lighten-5"),
+    "step-icon-bg": transparent,
+    "step-icon-color": map-get($grey, "lighten-5"),
+    "step-icon-border-color": rgba(map-get($grey, "lighten-2"), $alpha-3),
+    "step-line-color": rgba(map-get($grey, "lighten-2"), $alpha-3)
```

Comment:
These colors need to match the original values if the sass variables:
```
$step-icon-bg-visited: $base-inverse-color; // copy value from "base" section above
$step-icon-color-visited: map-get($grey, "darken-4");
$step-icon-bg: $base-inverse-color; // copy value from "base" section above
$step-icon-color: map-get($grey, "darken-4");
$step-icon-border-color: map-get($grey, "lighten-2");
$step-line-color: map-get($grey, "lighten-2");
```

might not need `step-icon-bg-visited` and `step-icon-bg` if those theme values can be replaced with `"base", "inverse-color"`